odxtools/dynamiclengthfield.py

```diff
@@ -56,4 +56,32 @@ def convert_physical_to_bytes(
         raise NotImplementedError()
 
     def decode_from_pdu(self, decode_state: DecodeState) -> ParameterValue:
-        raise NotImplementedError()
+
+        odxassert(not decode_state.cursor_bit_position,
+                  "No bit position can be specified for dynamic length fields!")
+
+        orig_origin = decode_state.origin_byte_position
```

Comment:
"DynamicLengthField: implement support for decoding" is a new feature, merits its own PR

Reply:
yeah, I took the patch out of the PR...